backup_and_restore/hibernating-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hibernating-cluster"]
+= Hibernating an {product-title} cluster
+include::_attributes/common-attributes.adoc[]
+:context: hibernating-cluster
+
+toc::[]
+
+You can hibernate your {product-title} cluster for up to 90 days.
+
+// About hibernating a cluster
+include::modules/hibernating-cluster-about.adoc[leveloffset=+1]
+
+[id="hibernating-cluster_prerequisites_{context}"]
+== Prerequisites
+
+* Take an xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backing-up-etcd-data_backup-etcd[etcd backup] prior to hibernating the cluster.
++
+[IMPORTANT]
+====
+It is important to take an etcd backup before hibernating so that your cluster can be restored if you encounter any issues when resuming the cluster.
+
+For example, the following conditions can cause the resumed cluster to malfunction:
+
+* etcd data corruption during hibernation
+* Node failure due to hardware
+* Network connectivity issues
+
+If your cluster fails to recover, follow the steps to xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore to a previous cluster state].
+====
+
+// Hibernating a cluster
+include::modules/hibernating-cluster-hibernate.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
```suggestion
[role="_additional-resources"]
.Additional resources
```

Reply:
I don't mind adding, but for some reason I thought this wasn't necessary anymore. Am I misremembering?